### DISCUSSION TITLE
quire build error

### DISCUSSION THREAD

**Discussion by kanotson** [thegetty/quire#684] [Get Help]
Hi, 
I'm having problems to get "quire build" working. Previewing works fine, but when I try to build the site, I get this error:

`[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] EPERM: operation not permitted, rename 'C:\Users\XXXX\Documents\XXXX\Quire\test1\_site' -> 'C:\Users\XXXX\Documents\XXXX\Quire\test1\.11ty-vite' (via Error)
[11ty]
[11ty] Original error stack trace: Error: EPERM: operation not permitted, rename 'C:\Users\XXXX\Documents\XXXX\Quire\test1\_site' -> 'C:\Users\XXXX\Documents\XXXX\Quire\test1\.11ty-vite'`

I've tried with 1.0.0-rc0, 1.0.0-rc3 and 1.0.0-rc5, with the same results. The OS is Windows10.

**Comment by Erin-Cecele** [comment]
Hi @kanotson. Thank you for reaching out. This is a known issue in Quire (see #626). We are working on a solution and will let you know when it is available.

**Reply by kanotson** [reply]
Thank you. I saw the issue about Windows 11 and seemed to me it was planned to be resolved in rc2, so I wasn't expecting to fail in rc5 too. Sorry if I've been redundant.

**Reply by Erin-Cecele** [reply]
Not a problem @kanotson. I apologize for not updating the note on that issue indicating the bug fix had been pushed back to a future release beyond rc2.

**Reply by Erin-Cecele** [reply]
Hi @kanotson apologies for the delay, this has been identified as https://github.com/11ty/eleventy-plugin-vite/issues/22, a possible fix seems to have been found, and there's hopefully a forthcoming PR to address it. We will update issue https://github.com/thegetty/quire/issues/626 when we have a fix. Thank you for your patience.

**Reply by Erin-Cecele** [reply]
Hi @kanotson, while the issue isn't fixed yet, I'm pleased to say we have discovered a workaround: https://github.com/thegetty/quire/issues/626#issuecomment-1542935869.
